board/technologic/ts7180/ts7180.c

```diff
@@ -118,30 +118,6 @@ void hw_watchdog_init(void)
 #endif
 }
 
-void hw_watchdog_reset(void)
```

Comment:
I notice this platform never actually seemed to call this, is it possible to get WDT functionality in U-Boot with these latest changes?

Reply:
WDT will be addressed later